src/runtime_src/tools/scripts/xrtdeps-win22.py

```diff
@@ -128,6 +129,15 @@ def main():
   if installLibraries( libraries, verbose) == True:
     return True
 
+  # -- Install python packages ----------------------------------------------
+  if args.python:
+    pybind = pybind11_package()
+    if not pybind.installed:
+      print("")
+      print("ERROR: The Python pybind11 package could not be installed")
+      print("        Please make sure you have pip3 installed on your system and try adding it with")
+      print("        py.exe -m pip install pybind11=='2.10.4'")
+
```

Comment:
An error condition was encountered.  As such, this method should return a True value.
```suggestion
     return True
```

Reply:
Done